core/aws_ddk_core/stages/glue_transform.py

```diff
@@ -138,15 +144,15 @@ def __init__(
             result_path=JsonPath.DISCARD,
         )
         # Create start crawler step function task
-        crawl_object = CustomState(
```

Comment:
Didn't we lose this `Catch` condition now?

I think you'll need to do:
```python
crawl_object.add_catch(Succeed("success"), errors=["Glue.CrawlerRunningException"])
```

Reply:
Good _**catch**_

I used `Fail` instead of `Succeed` though.